templates/debug.php

```diff
@@ -714,6 +748,15 @@ function stopCountdownManually() {
         </table>
     <?php endif ?>
 <?php endforeach ?>
+<?php
+	$page_params = array(
+		'title_tag'      => 'h2',
+		'display_toggle' => true,
+		'is_open'        => false,
+	);
+
+	fs_require_template( 'debug/scheduled-crons.php', $page_params );

```

Comment:
@DanieleAlessandra Regarding the duplicated `<button class="toggle-table" aria-expanded="true">`, if it's stored in a variable, then it can also be included in `$page_params`.